app/assets/javascripts/fortitude/jquery/blocks/flashbar.js

```diff
@@ -1,13 +1,12 @@
 (function($) {
     'use strict';
 
-    $(document).
-      on('close:ft:flashbar', '.flashbar', function() {
-        $(this).remove();
-      }).
-      on('click.ft.flashbar.data-api', '.flashbar__close', function(event) {
-        $(this).closest('.flashbar').trigger('close:ft:flashbar');
-        event.preventDefault();
-      });
+    $(document).on('close.ft.flashbar', '.flashbar', function(event) {
+      $(this).remove();
+    });
+
+    $(document).on('click', '.flashbar__close', function(event){
+      $(this).closest('.flashbar').trigger('close.ft.flashbar');
+    });
 
```

Comment:
need $this.data().showClass and $this.data().hideClass that use a simpler version of waitForAnimation so it only hides/shows if the animation is finished.